source/_components/egardia.markdown

```diff
@@ -0,0 +1,67 @@
+
+To enable the integration with your alarm panel, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+egardia:
+  host: YOUR_HOST
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+Configuration variables:
+
+- **host** (*Required*): The local IP address of the Egardia/Woonveilig alarm panel.
+- **username** (*Required*): Username for the Egardia/Woonveilig account.
+- **password** (*Required*): Password for Egardia/Woonveilig account.
+- **version** (*Optional*): The version of the Egardia system. `GATE-01`, `GATE-02` and `GATE-03` are currently supported. Defaults to `GATE-01`.
+- **port** (*Optional*): The port of the alarm panel. Defaults to 80.
+- **report_server_enabled** (*Optional*): Enable reporting by server. Defaults to `False`.
+- **report_server_port** (*Optional*): Port of the Egardia server. Defaults to 52010.
+- **report_server_codes** list (*Optional*): List of codes for the different states.
+
+Note that this basic configuration will only enable you to read the armed/armed away/disarmed status of your alarm and will **not** update the status if the alarm is triggered. This is because of how Egardia built their system. The alarm triggers normally go through their servers.
+You can change this, however, using the following procedure. This is a more advanced configuration.
+
+1. Log in into your alarm system's control panel. You will need to access http://[IP of your control panel]. You know this already since you need it in the basic configuration from above. Log in to the control panel with your Egardia/Woonveilig username and password.
+2. Once logged in, go to *System Settings*, *Report* and change the Server Address for your primary server to the IP or hostname of your Home Assistant machine. You can leave the port number set to 52010 or change it to anything you like. **Make sure to change the settings of the primary server otherwise the messages will not come through. Note that this will limit (or fully stop) the number of alarm messages you will get through Egardia's / Woonveilig services.** Maybe, that is just what you want. Make sure to save your settings by selecting 'OK'.
+3. On your Home Assistant machine run `$ sudo python3 egardiaserver.py`. Refer to the [python-egardia repository](https://github.com/jeroenterheerdt/python-egardia) for detailed documentation on parameters. This will receive status codes from your alarm control panel and display them. You will need the codes to include in your configuration.yaml. Make sure to change the status of your alarm to all states (disarm, arm, home) by all means possible (all users, remotes, web login, app) as well as trigger the alarm in all ways possible to get 100% coverage. **Before triggering the alarm it might be good to disable the siren temporarily (can be done in Panel Settings).**
```

Comment:
This step needs updating since we now start the server from home assistant.

Reply:
I do not agree - the `egardiaserver.py` script serves in this step to capture alarm codes. The user is asked here to run the script and capture the codes so they can be put into the config.

Reply:
Ok, then this needs to be clarified. Eg, for how long does the user need to have the script running. I'm guessing the user needs to actively shut the server down, by exiting the script.

Reply:
Fair enough, I tried to make it more clear. Hope this helps.